query-helpers/profile.js

```diff
@@ -21,7 +21,7 @@ function isAggregate(ids) {
   return ids.length > 1;
 }
 
-const profileSQL = (profile, ids, isPrevious = false) => `
+const profileSQL = (ids, isPrevious = false) => `
```

Comment:
Isn't `query-helpers/acs.js` replacing this file? Can't this file be deleted?

Reply:
Nit: Prefer that the file rename command is used in git to track the file renaming.